app/views/service_providers/_form.html.erb

```diff
@@ -1,20 +1,20 @@
 <%= form.error_notification %>
 <fieldset class='usa-fieldset'>
 
-  <legend class='usa-sr-only'>Production Configuration</legend>
+  <legend class='usa-sr-only'>Dashboard Configuration</legend>
   <%= form.label :prod_config,
-                 label: 'Production Configuration',
-                 class: 'usa-label',
-                 hint: I18n.t('service_provider_form.prod_config').html_safe %>
+                 label: 'Dashboard Configuration',
+                 class: 'usa-label' %>
   <%= form.collection_radio_buttons(:prod_config,
-                                    { "Yes" => true, "No" => false },
-                                    :last, :first,
-                                    item_wrapper_tag: :div,
-                                    item_wrapper_class: "usa-radio") do |b| %>
-    <%=  b.radio_button(class: "usa-radio__input") %>
-    <%=  b.label(class: "usa-radio__label")%>
+    { "Production" => true, "Sandbox" => false },
+    :last, :first,
```

Comment:
hm i feel like the formatting difference is wigging me out a little. this is non blocking, but could you make it consistent? (i thought rubocop dealt with this, but maybe not? maybe we should tighten the rules)

Reply:
honestly i prefer the way you're indenting, but it would be real out of scope to fix the whole view, so i dunno

Reply:
I can fix that.
  For the record, this indentation makes me very uncomfortable. I'd much rather we do:
```
<% some_function(
  :param1,
  :param2,
  param3: true
) do |var| %>
```